src/Input/TextField/TextField.test.tsx

```diff
@@ -146,3 +146,31 @@ describe('when removeFloatingLabel is true', () => {
     expect(textFieldInput).toBeTruthy();
   });
 });
+
+describe('<TextField /> forwards ref to underlying input element', () => {
+  const label = 'forward-ref';
+
+  test('ref is being forwarded correctly', () => {
+    const ref = React.createRef<HTMLInputElement>();
+    const { queryByLabelText } = render(
+      <TextField ref={ref} label={label} type="text" />
+    );
+    const textInput = queryByLabelText(label);
```

Comment:
Can reuse the function `setupTextField`  at line 19 as below snippet

```
describe('<TextField /> forwards ref to underlying input element', () => {
  const ref = React.createRef<HTMLInputElement>();

  test('ref is being forwarded correctly', () => {
    const { textFieldInput } = setupTextField({ ref });
    expect(textFieldInput).toEqual(ref.current);
  });
 // ...
});
```

Reply:
hmm, I think we should not expect those functions will be reused. Because it's not reasonable to review the whole test file before adding new test cases. What do you think?

Reply:
another point is that writing test code and writing production code are different. In general, we want to be DRY when writing production code. But when writing test code, DRY is not that necessary in my opinion.

Reply:
Yeah, I agree with you, DRY is not that necessary for writing testing.
But I think it is also good to apply for readability, I think we can consider limiting the reusable function in `describe` scope and depend on how many repetitive codes are there.

Reply:
I am not sure if I understand you correctly, which way has better readability?

I think `const { textFieldInput } = setupTextField({ ref });` is less readable, especially when we are reviewing diffs, we cannot know what `setupTextField` does unless we review the whole file to find its implementation.